src/components/Headbottom.js

```diff
@@ -0,0 +1,72 @@
+function Headbottom() {
+  return (
+    <div className="Headbottom">
+      <span>
+        <img
+          id="imgOne"
+          src="https://education-team-2020.s3.eu-west-1.amazonaws.com/web-dev/labs/landing-page/icon1.png"
+          alt="Declarative"
+          width="150px"
+        />
+        <h2>Declarative</h2>
+        <p id="paragraphHbOne">
+          React makes it <br />
+          painless to create
+          <br />
+          interactive UIs
+        </p>
+      </span>
```

Comment:
You should create a new component, every time you see repeated code, and then just call it when you need it.